src/Tribe/Tickets.php

```diff
@@ -245,9 +246,9 @@ public function get_ticket_move_url( $post_id, $ticket = null ) {
 		 * @since TBD
 		 *
 		 * @param int $post_id the id of the parent post/event
-		 * @param obj $ticket Ticket object
+		 * @param object $ticket Ticket object
 		 *
-		 * @return string HTML link
+		 * @return string HTML link | void
```

Comment:
Should be `@return string|void HTML link` (following the pattern tag - type(s) - description)